doc/installation.rst

```diff
@@ -9,43 +9,45 @@ Installation
    :local:
    :backlinks: top
 
-Platforms
----------
+Supported Platforms
+-------------------
+
+*SfePy* is known to work on various flavors of Linux, Intel-based MacOS and
+Windows. It can work with both Python 2.7 and Python 3.x.
 
-*SfePy* is known to work on various flavors of Linux, on Intel Macs and
-Windows. It can work with both Python 2.7 and Python 3. Depending on Python
-installation and OS used, replacing ``python`` by ``python3`` might be required
-in all the commands below (e.g. in :ref:`compilation`) in order to use Python 3.
+Depending on Python installation and OS used, replacing ``python`` by ``python3`` might be required
+in all the commands below (e.g. in :ref:`compilation`) in order to use Python 3. However we strictly recommend to use *Python Virtual Environments* (see `Python virtualenv`_ or `Conda Environments`_) for these purposes.
 
```

Comment:
Yes, I would just mention anaconda, so that it does not look that one must use a virtualenv. In practice, there is IMHO no mess - people have either Python 2.7, or 3.6. People having more versions are usually developers that know what they are doing, and do not need to be instructed.